cmd/configschema/README.md

```diff
@@ -1,5 +0,0 @@
-# ConfigSchema API
```

Comment:
Should we still keep some README here (something like "this folder contains config schema related tooling")?